lib/pages/animation/flipbook.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:flutter/material.dart';
+import 'flip_item.dart';
+
+/// そもそもStackとは？
+/// -> ウィジェットを重ねることができるウィジェット (https://flutter.ctrnost.com/basic/layout/stack/)
+///    Stackを使ったサンプルページ
+/// 　　指定した時間を経過するとStackの子要素として設定しているウィジェットを非表示にしてみる
+class Flipbook extends StatelessWidget {
+  const Flipbook({super.key});
+
+  /// Stackとして重ねる画像
+  final items = const <FlipItem>[
+    FlipItem(
+      imagePath: 'assets/images/10.png',
```

Comment:
```suggestion
      imagePath: 'assets/images/animation/10.png',
```
画像パスはこちらですかね！
`pubspec.yaml` に assets の追加が必要かもなのでご確認お願いします！

Reply:
そうです！それで動かなくなってることにあとから気づいて、記事書いたけど、こっち直してなかった、ごめんなさい。images内にanimationフォルダを作りました。